ReactWindows/ReactNative.Shared/UIManager/BorderedCanvas.cs

```diff
@@ -14,7 +12,7 @@ namespace ReactNative.UIManager
     /// <summary>
     /// Represents a Canvas with an optional Border inside.
     /// </summary>
-    public class BorderedCanvas : Canvas
+    public class BorderedCanvas : Canvas, IAccessible
```

Comment:
Why is this being implemented for bordered canvas? I thought button is always going to result in ReactControl. Is there anything else that will result in BorderedCanvas?

Reply:
from "accessible invocation" point of view the fact that all invokable controls will be wrapped in ReactControl and the ReactControl will mirror the onPress of the original control on itself is not guranteed. So I thought that we should adhere to API which says View can have onPress, and not to current implementation that is not guaranteed to not unexpectedly change in future.

Second point is that this is not restricted to trait only - it's the general design to implement anything in accessibility that requires custom automation peer.

---
In reply to: [168883927](https://github.com/gmangov/react-native-windows/pull/1#discussion_r168883927) [](ancestors = 168883927)